pkg_defs/ska3-flight/meta.yaml

```diff
@@ -1,6 +1,6 @@
 package:
   name: ska3-flight
-  version: 2020.02.12
+  version: 2020.04.24
```

Comment:
I forget if there was some agreement or reasoning otherwise, but I thought this should just be 2020.4.  That makes the version here match the release tag and the milestone.

Reply:
In other words, starting in 2020 all the version tags will be YYYY.N where N just increases by 1 for each release.  This can include skipped releases.  That would include the others like ska3-template or ska3-core.

Reply:
My disagreement was that when these were candidates I lost my ability to make packages with unique names as we went through updates.  For example, in this PRI already would have incremented to 04.25 after the mica fix because knowing me I'd have built 04.24 without mica and would have that actual packagehanging around to confuse me (or my auto package building wouldn't have actually rebuilt ska3-flight after adding mica because 04.24 already existed).

I don't really care if we move forward with some of @javier's ideas for SCM of the meta.

Reply:
I don't mind either way. I also thought that it would be 2020.4, but in 2020.3 I saw Jean was still following the date format and that got me thinking that releases can be 2020.n until the actual final tag, and then they get their final "date" name. But that was just me thinking, I agree.

So... no problem in changing it to 2020.4

Reply:
I have certainly experienced the pain of "over-caching" by conda and getting confused.  So we might consider using a standard versioning convention like `2020.4rc<N>` until the last one.  I can't tell if the pain of that is greater than the pain of remembering to clear caches or force-build.

Reply:
yes, having release candidate versions labeled -rc# is what I had in mind. I worry then that at each step I have to modify the meta.yaml to set the corresponding version, and after all testing remove the -rc#. Ideally, this would come from the tag only.

Reply:
And it gets worse if you are changing ska3-core, so any update there drives an update to ska3-flight as well. Definitely :+1: if you can find an elegant SCM way out of this mess. I also don't know if the conda build version can help us out.

Reply:
FWIW, the Python standard https://www.python.org/dev/peps/pep-0440/#pre-releases does not include a dash before the `rc`.

Reply:
I had wondered if it would be less work in the end to just manage ska3-flight and ska3-core as two new *very small* new git repos but I think @javierggt has better ideas.

Reply:
We also talked about maybe just combining ska3-core into ska3-flight but I have enjoyed having noarch updates possible for ska3-flight (though that could all be moot with real multiplatform auto builds).

Reply:
What I have in mind at this time is the following:
- One releases one meta-package at a time, so ska3-core would have to be released before ska3-flight. Each ska3-flight corresponds to one and only one ska3-core. There is no issue. 
- When you make a release (or just a tag) an action is triggered. This action reads meta.yaml and overwrites the name of the package to create a conda package which is uploaded to a repository. This is the same as with other packages, except the name overwriting.
- this package eventually winds its way to the test conda channel (instead of masters).
- when a new ska3-flight package appears in the test conda channel, that triggers a testr run.
- eventually the results from this run make it to the dashboard (steps omitted here).
- Once we agree on a final yaml config, we tag/release again, without the rc#. The process repeats, and the package ends up wherever we want it.

Reply:
I'm not quite sure if we can promote ska3-core updates independent of ska3-flight, but we could probably release in that order?  Not sure where to capture this discussion but should probably resolve the conversation for this line of the meta.yaml :-)